Environmental-Test-Chamber/enviornmentalTestChamberPDD.tex

```diff
@@ -0,0 +1,279 @@
+
+Some examples of near-term testing benefits include HAB camera shutdown investigation. HAB-CV camera performance, IREC desert heat validation, and \$50Sat orbital thermal cycling. Sensors and electronics have performance curves related to environmental conditions. An ETC would enable fine characterization of these sensors before flight, increasing confidence in the data collected.\@
+
+An ETC would have reduced mission risk for each of those projects and will be useful for future iterations of those types of projects. An ETC would likely prove useful to many other applications in the near to medium future.
+
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+  At the end of the project, a working Environmental Test Chamber should be delivered. It should be capable of maintaining a near-vacuum for extended periods of time, as well as cycling between standard and vacuum-like pressures at reasonable rates.
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+  The chamber should also be able to subject the test article to thermal variations between -20 C and 50 C. Cooling is considered an advanced feature due to the difficulty of removing thermal energy from the ETC compared to adding thermal energy. 
+
+  An additional feature would be the ability to direct the applied thermal energy to simulate satellite tumbling or orbital cycles. Another metric to consider is the ability to apply the average solar radiation ( 1,368 W/m2) to the working surface of the test article to simulate on-orbit thermal conditions.
+
+  Along with the ETC hardware, there should be a control system allowing for software control of pressure and temperature, as well as data recording and real-time monitoring of the current environment. Users should be able to define preset environmental conditions that the ETC should attempt to maintain within reasonable bounds. Also users should be able to define a test sequence, which may consist of a number of environmental conditions varying over time, as well as the ability to cycle between high and low values for controlled environmental conditions. These should be repeatable N times or for a preset time period.
+
+  Finally, detailed operating instructions should be written and validated, to provide a safe way for other members to utilize the ETC. Instructions on setup, operation, maintenance, as well as detailed safety procedures in case of an accident or malfunction should be written and tested with the final hardware.
+
+\subsection{Milestones}
```

Comment:
As much as I hate bulleted lists, it might be applicable here. I recommend a table as always, but still. Plain indented paragraphs look bad.